app/assets/javascripts/application.js

```diff
@@ -0,0 +1,3 @@
+//= require jquery
+//= require jquery_ujs
```

Comment:
I've removed them. I believe I was using them to check jasmine tests in the browser, but they are redundant, I agree.